src/http-auth-interceptor.js

```diff
@@ -122,7 +122,10 @@
        */
       retryAll: function(updater) {
         for (var i = 0; i < buffer.length; ++i) {
-          retryHttpRequest(updater(buffer[i].config), buffer[i].deferred);
+          var _cfg = updater(buffer[i].config);
+          if (_cfg === false)
```

Comment:
That is exactly the case. Some of the request are supposed to be retried,
others to be canceled.
29 lut 2016 14:35 "Witold Szczerba" <notifications@github.com> napisał(a):

> In src/http-auth-interceptor.js
> <https://github.com/witoldsz/angular-http-auth/pull/127#discussion_r54409370>
> :
>
> > @@ -122,7 +122,10 @@
> >         */
> >        retryAll: function(updater) {
> >          for (var i = 0; i < buffer.length; ++i) {
> > -          retryHttpRequest(updater(buffer[i].config), buffer[i].deferred);
> > +          var _cfg = updater(buffer[i].config);
> > +          if (_cfg === false)
>
> Why won't you call rejectAll instead? Is is that there are some events
> you want to abandon and some you want to be retried? What's the case?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/witoldsz/angular-http-auth/pull/127/files#r54409370>.
>